sync/sync.go

```diff
@@ -687,10 +687,10 @@ func MakeBranchPRs(f *Flags, dir string, entry *ConfigEntry) ([]SyncResult, erro
 		if force {
 			c.Args = append(c.Args, "--force")
 		}
+
 		c.Args = append(c.Args, auther.InsertAuth(remote))
-		for _, r := range refspecs {
-			c.Args = append(c.Args, r)
-		}
+		c.Args = append(c.Args, refspecs...)
+
```

Comment:
Also doesn't seem like gofumpt.

Reply:
few linter errors are there I fixed with this PR.